docker-compose.yml

```diff
@@ -1,5 +1,9 @@
 version: "3.7"
 
+networks:
+  test-network:
+    external: true
+
```

Comment:
This network can be renamed.
I still have to check that both the indexer and scraper work fine using this network.
Indexer draft PR configuring the network: https://github.com/across-protocol/indexer/pull/142